benchmark_v2/parquet-fixed-strings_benchmark.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+
+import pytest
+
+import arkouda as ak
+
+
+@pytest.mark.skip_numpy(True)
+@pytest.mark.skip_correctness_only(True)
+@pytest.mark.benchmark(group="Parquet_Fixed_Strings")
+@pytest.mark.parametrize("scaling", [False, True])
+@pytest.mark.parametrize("fixed_size", [1, 8])
+@pytest.mark.parametrize("nfiles", [1])
```

Comment:
Please reconcile with the PR message that claims:
*  nfiles: number of files (1, 5, 10) [I see only `1`; if the other options are added, this should probably be incorporated into `extra_info["description"]`]
*  scaling: scale down file size per shard (True/False) [this matches]
*  fixed: use fixed_len when reading Parquet files (True/False) [instead I see `fixed_size`]